edx_exams/apps/core/migrations/0008_alter_exam_exam_type.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.2.14 on 2022-09-01 19:39
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('core', '0007_alter_proctoringprovider_name'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='exam',
+            name='exam_type',
+            field=models.CharField(choices=[('proctored', 'Non-practice, timed, proctored exam'), ('timed', 'Non-practice, non-proctored, timed exam'), ('practice', 'Practice, non-proctored, timed exam'), ('onboarding', 'Practice, timed, proctored exam')], db_index=True, max_length=255),
```

Comment:
Nit: For some of the choices, timed comes before proctored and for others after:
'Non-practice, timed, proctored exam'
'Non-practice, non-proctored, timed exam'
For readability, you could consider having the order be consistent for these strings.